### ISSUE TITLE
Managed connectors not displayed for the provider company

### ISSUE THREAD

**Issue by gitbooma** [eclipse-tractusx/portal-backend#1170]
### Current Behavior
With 24.05 managed connector details look incorrect
Company 1: e2e test
Company 2: tect-ch

connector name: test-con.mgd
  provider: e2e test
  host: test-ch

From 'test-ch' company, on Connector Registration screen, managed connectors page:
![image](https://github.com/user-attachments/assets/9695c34f-d660-42f9-8a1d-eb97963d2251)

issues:
- Provider company is shown under 'Customer', 
- In 'e2e test' the Connector Registration screen does not display this connector(test-con.mgd).

### Expected Behavior
Provider company should see the managed connectors that their customers host.
Provider company should not be displayed under 'customer'

### Steps To Reproduce
- onboard 2 companies, connector-provider and host
- onboard an app from the company 'connector-provider' and subscribe to the app from the company 'host'
- once the subscription is active, from 'connector-provider'  register a managed connector for this subscription
- once the registration is done, **the connector is not shown in 'connector-provider'**
- The connector is displayed in 'host' company's managed conenctor page and **the 'connector-provider' is displayed as 'customer'**

**Comment by Phil91** [comment]
@evegufy yes can confirm that for some reason the customer isn't able see the details

**Comment by evegufy** [comment]
@dhiren-singh-007 @gitbooma could you please provide a status?

**Comment by dhiren-singh-007** [comment]
Once this enahncement eclipse-tractusx/portal-backend#1087 is done then we can mark this as closed

**Comment by saadanzari** [comment]
@dhiren-singh-007 pls add the feature from sig-release.
